spinaltap-mysql/src/main/java/com/airbnb/spinaltap/mysql/event/mapper/MysqlMutationMapper.java

```diff
@@ -63,7 +63,9 @@
     return ClassBasedMapper.<BinlogEvent, List<? extends Mutation<?>>>builder()
         .addMapper(TableMapEvent.class, new TableMapMapper(tableCache))
         .addMapper(GTIDEvent.class, new GTIDMapper(gtid))
-        .addMapper(QueryEvent.class, new QueryMapper(beginTransaction, gtid, schemaTracker))
+        .addMapper(
```

Comment:
`lastTransaction` also needs to be updated if there's a DDL statement in QueryEvent.